src/client/public/scripts/dash/dash.ts

```diff
@@ -414,6 +475,7 @@ async function start() {
     changeFavIcon(false);
 
     lastViewedData = lastPolledData;
+    lastActivityUpdateTimestamp = Date.now();
```

Comment:
So is the timestamp from the server ignored now? If so, should we just remove it from there?

Reply:
The two diff timestamps make sense, but with your changes, it might make sense to just remove server timestamp and use local time instead.

Reply:
Circling back on this.

The timestamps from the server are still needed.

We short poll our server and ask "what was the last known update (via github hook) that we heard about (using the servers timestamp)" and then we compare that to the last timestamp that we hard for a full dash load".

The local activity is, when has it been longer than an hour with the dashboard open. This is seperate form the server time.